### DISCUSSION TITLE
Firefox (multiprocess) memory usage not reflected correctly

### DISCUSSION THREAD

**Discussion by heinrich5991** [nicolargo/glances#2523] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I like to check on what processes hog memory when trying to launch a memory-intensive program. Unfortunately, I can't really check firefox's memory usage because it's split across tens of processes.

**Describe the solution you'd like**
A way to group processes by their name and add their resource requirements. Alternatively, even better, doing this automatically for known multi-process applications.

**Comment by nicolargo** [comment, marked as answer]
Hi @heinrich5991 

click on 'j' hotkey (or use the --program option on the command line) to group processes by programs instead of threads.

Nicolas

Source: https://glances.readthedocs.io/en/latest/aoa/ps.html
